Implementacion de Dashboard de Pedidos.

lomandopuntocom opened the pull request:
![Screenshot 2025-04-07 171147](https://github.com/user-attachments/assets/f59a24fe-c4f7-487f-811c-71eeb332588e)
Implementación del dashboard de pedidos más instalación de Tailwind y Router

lomandopuntocom committed:
Implementacion de Dashboard de Pedidos.

Implementación del dashboard de pedidos más instalación de Tailwind y Router

Co-Authored-By: RaidenMT <23229026+RaidenMT@users.noreply.github.com>

lomandopuntocom commented:
@RaidenMT había olvidado etiquetarlo, perdon

lomandopuntocom committed:
Borrado Flowbite por completo

RaidenMT commented on OrderNow-Dashboard/.vscode/extensions.json:
No subir ese archivo

RaidenMT commented on OrderNow-Dashboard/src/App.jsx:
```diff
@@ -1,35 +1,18 @@
+    <Routes>
+      <Route path="/" element={<Home />} />
+      <Route path="/about" element={<About />} />
+      <Route path="*" element={<NotFound />} />
```
Quitar esta ruta por el momento

RaidenMT commented on OrderNow-Dashboard/src/App.jsx:
```diff
@@ -1,35 +1,18 @@
-    </>
-  )
+    <Routes>
+      <Route path="/" element={<Home />} />
```
Eliminar esta ruta

RaidenMT commented on OrderNow-Dashboard/src/pages/Home.jsx:
Eliminar archivo

RaidenMT commented on OrderNow-Dashboard/src/pages/OrderDashboard.jsx:
```diff
@@ -0,0 +1,137 @@
+
+  useEffect(() => {
+    setTimeout(() => {
+      const mockOrders = [
```
Cambiarlo a un archivo json para leer desde ahí

RaidenMT commented on OrderNow-Dashboard/src/pages/OrderDashboard.jsx:
```diff
@@ -0,0 +1,137 @@
+
+      setOrders(mockOrders);
+      setLoading(false);
+    }, 1000);
```
Por qué tenemos un setTimeout?

lomandopuntocom replied:
era para simular que se cargaba desde una api real.
en seguida lo quito

RaidenMT commented on OrderNow-Dashboard/src/pages/OrderDashboard.jsx:
```diff
@@ -0,0 +1,137 @@
+
+  const getStatusText = (id) => {
+    switch (id) {
+      case 1: return '🟡 Pendiente';
```
Traer desde Base de datos o justificar por qué se hace esto

lomandopuntocom replied:
Conectaré la base de datos en el siguiente PR, esto se cambiará en el futuro.

RaidenMT commented on OrderNow-Dashboard/src/pages/OrderDashboard.jsx:
```diff
@@ -0,0 +1,137 @@
+    }
+  };
+
+  const formatDate = (dateStr) => {
```
Buena función, pasarla a otro archivo que tengas funciones de utilidades para formatear fechas

RaidenMT commented on OrderNow-Dashboard/src/pages/OrderDashboard.jsx:
```diff
@@ -0,0 +1,137 @@
+    <div className="min-h-screen p-6 md:p-12 font-sans">
+      <div className="max-w-7xl mx-auto">
+        <h1 className="text-3xl font-bold text-center text-gray-800 mb-8">
+          📋 Pedidos del Restaurante
```
Quitar ícono

RaidenMT reviewed (changes requested):
Revisar los comentarios realizados por favor

lomandopuntocom committed:
Implementacion de Cambios

Movidos los datos falsos a un .json.
eliminados home.jsx y NotFound.jsx.
Funcion para formato de fecha movido a carpeta 'utils'

RaidenMT commented on OrderNow-Dashboard/.vscode/extensions.json:
Quitar archivo

RaidenMT commented on OrderNow-Dashboard/src/pages/OrderDashboard.jsx:
```diff
@@ -0,0 +1,70 @@
+import React, { useEffect, useState } from 'react';
+import mockOrders from '../fakeData/mockOrders.json' // Ajusta la ruta si es necesario
```
Eliminar comentarios de chatgpt

RaidenMT reviewed (changes requested):
Tiene comentarios por resolverse todavía

lomandopuntocom committed:
Arreglos finales

removido archivo extensions.json y comentarios de chat gpt

RaidenMT reviewed (changes requested):
Falta un último comentario por resolver.

lomandopuntocom commented:
> Falta un último comentario por resolver.

cual inge? ya repase todos y no veo que falta

lomandopuntocom committed:
Remover Iconos

Se removieron los iconos de los estados
